jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile

```diff
@@ -14,5 +14,5 @@ rollingUpgradePipeline(
     workaround_kernel_bug_for_iotune: false,
     internode_compression: 'all',
 
-    timeout: [time: 360, unit: 'MINUTES']
+    timeout: [time: 550, unit: 'MINUTES']
```

Comment:
why are you adding extra 3 hours to this job?

Reply:
Most of times it was run on centos7. On this os Rolling upgrade run a bit longer.